retrieval_service/datastore/providers/neo4j_graph.py

```diff
@@ -169,20 +196,62 @@ async def search_airports(
     async def get_amenity(self, id: int) -> Optional[models.Amenity]:
         async with self.__driver.session() as session:
             result = await session.run(
-                "MATCH (amenity: Amenity {id: $id}) RETURN amenity", id=id
+                """
+                MATCH (amenity: Amenity {id: $id})
+                RETURN amenity.id AS id,
+                    amenity.name AS name,
+                    amenity.description AS description,
+                    amenity.location AS location,
+                    amenity.terminal AS terminal,
+                    amenity.category AS category,
+                    amenity.hour AS hour
+                """,
+                id=id,
```

Comment:
specify return fields so embedding wont be included in it

Reply:
Can you add this as a comment?